portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsParagraphField.testcase

```diff
@@ -231,12 +231,46 @@ definition {
 	}
 
 	@description = "Verify that a Table can be created in the Body Text"
-	@ignore = "true"
 	@priority = "4"
 	test CanSupportTables {
+		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131236 CanSupportTables pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		Click(
+			key_titleName = "Table",
+			locator1 = "FormFields#TOOLBAR_ANY_ITEM_BUTTON");
+
+		Click(locator1 = "CKEditor#OK_BUTTON");
+
+		Form.save();
+
+		AssertElementPresent(locator1 = "Message#SUCCESS");
+
+		ProductMenu.gotoPortlet(
```

Comment:
@piresdm I've changed this to `Click(locator1 = "Icon#BACK");` to reduce the UI navigation executed by the test.
Please see: https://github.com/emanuelfisilva/liferay-portal/pull/167/commits/0be51463ddd0bc4feb957deb4445efefb1e3e4ec